modules/ROOT/pages/errors/all-errors.adoc

```diff
@@ -1,8 +1,27 @@
-:description: The Neo4j error codes for Neo4j version {neo4j-version}.
+:description: The error codes for Neo4j 5.
 
 
 [[neo4j-errors]]
-= List of all server error codes
+= List of Neo4j error codes
+
+This page lists the current Neo4j error codes, which will be replaced by xref:errors/gql-errors.adoc[GQLSTATUS error codes] in the future.
+
+Error codes are returned by the server when the execution of a query fails.
+They always have the severity level `ERROR`.
+Errors are grouped based on the type of the Neo4j code:
+
+Client errors::
+These errors are caused by the client (user input or user application) and are usually related to the request itself.
```

Comment:
You may want to double check with @tobias-johansson. He recently looked into the classification. One source of some client errors are configuration setting, neo4j.conf setting, or such. In that case, the request as such is in principle fine but does not fit the configuration. I do not know if he ended up including these, I'd guess so. If yes, it may be worthwhile mentioning the "configuration" here to.

Reply:
I'll keep this in mind for when it is implemented and update this section.

Reply:
By the way, this is the old category. I'll add this new classification to the GQLSTATUS errors and comment it out until it is implemented.